docker/rundocker.sh

```diff
@@ -3,7 +3,7 @@
 # A wrapper script to run viral-ngs docker images
 # The following paths have to be modified according to end-user environment
 NOVOALIGN_PATH="/opt/novocraft" # Directory where novoalign.lic license file can befound
-GATK_PATH="/opt/GenomeAnalysisTK-3.6" # Directory where the correct GATK jar file can be found
+GATK_PATH="/opt/GenomeAnalysisTK-3.8" # Directory where the correct GATK jar file can be found
```

Comment:
@tomkinsc add this to https://github.com/broadinstitute/viral-ngs/blob/master/DEVELOPMENT_NOTES.md#upgrading-gatk
?

Reply:
done, thanks

Reply:
Has anyone used rundocker.sh recently? It doesn’t get used for anything more pipelined, but for ad hoc command line work, is it useful beyond what you get from the usual “docker run -it —rm”?

Reply:
I haven't used it recently.